### DISCUSSION TITLE
Is it possible to set a size to display 'Single Layout' components?

### DISCUSSION THREAD

**Discussion by jowilhnson** [histoire-dev/histoire#477] [Q&A]
When viewing my components in histoire, anything that uses the default Single Layout displays 731 x 155.8.
This means I usually have to resize the view every time I look at something. 
I have not been able to find where this display size is being set, or any way to override it.
Even if we could set it to use one of the responsive presets by default, that would be better.
Automatically filling the display area would be even better.

**Comment by kiddicodes** [comment]
Could this be what you are looking for?
https://histoire.dev/reference/vue3/variant.html#responsivedisabled

It disables the responsive handles completely and "always fit the available space"

If you do want to change the presets, there is a setting for that as well:
https://histoire.dev/reference/vue3/variant.html#responsivedisabled

Hope this helps you 😊
